backend/danswer/db/chat.py

```diff
@@ -1,3 +1,4 @@
+import datetime
```

Comment:
Slight preference for having this in a separate module like usage_export.py or something